sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -45,11 +45,30 @@ Follow the guide to
     ```
 1. After you've created the roles go to your Enterprise Application in Azure Portal, select "Users and groups" and assign the new roles to your Users (assignment of roles to groups is not available in the free tier of AAD).
 
+### Configure access other resources server
+This is an optional configuration. This guide is for accessing [Resource Server Obo].
+If you want to use **webapp** to access other resource server (for example, access [Resource Server Obo] or [Resource Server] or custom resource server), you can refer to this guide.
+
+1. First you need to complete [Configure obo for resource server] and make sure to expose the scope of `Obo.WebApiA.ExampleScope`.
```

Comment:
`Configure obo for resource server`  -> `Config for resource server obo`. And it's better to use lower case: `Config` -> `config`.